### PR TITLE
Use css overrides instead of css vars, various other small fixes

### PR DESCRIPTION
**Pull request by mikehenrty**

### CODE REVIEW

File: app/js/controller/list_controller.js [hunk 1]
```diff
@@ -165,20 +165,27 @@ export default class ListController extends Controller {
...
+          break;
+
+        case 'MANIFEST_URL_ERROR':
+          errorMsg = 'Invalid manifest url';
           break;
 
-        default:
-          this.showAlertDialog('Error installing: ' + err.target.error.name);
+        case 'INVALID_SECURITY_LEVEL':
+          errorMsg = 'Invalid permissions';
```
Comment (DouglasSherk) [inline app/js/controller/list_controller.js]: We should probably display something about enabling developer mode here. That can be a follow-up, though.

File: app/js/controller/list_controller.js [hunk 2]
```diff
@@ -165,20 +165,27 @@ export default class ListController extends Controller {
     }
 
     installReq.onerror = (err) => {
+      var errorMsg = err.target.error.name;
       var errorType = err.target.error.name;
```
Comment (DouglasSherk) [inline app/js/controller/list_controller.js:169]: Why have this twice?

File: bower.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     "gaia-theme": "gaia-components/gaia-theme#0.4.5",
```
Comment (DouglasSherk) [inline bower.json:11]: Might want to generally go through these and update them. I think some of them are getting a bit out of date.

File: app/js/controller/list_controller.js (hunk 2 above)
Comment (mikehenrty) [inline app/js/controller/list_controller.js:169]: You're right, this is kinda confusing. The point is that `err.target.error.name` is the default error message, but that's not clear here. I'll restructure.

File: bower.json (hunk 3 above)
Comment (mikehenrty) [inline bower.json:11]: I'll update those in a followup.

File: app/js/controller/list_controller.js (hunk 1 above)
Comment (mikehenrty) [inline app/js/controller/list_controller.js]: I can add this now.

**Pull request merged**